data/administrations/ademe.yml

```diff
@@ -0,0 +1,16 @@
+slug: ademe
+short: ADEME
+site: https://www.ademe.fr/
+logoType: portrait
+long: Agence de la transition écologique
+contact: https://www.ademe.fr/contact/
+apiMonitors:
+  - apiName: API Professionnels RGE
+    apiGouvLink: https://api.gouv.fr/les-api/api_professionnels_rge
+    dataGouvLink: https://www.data.gouv.fr/fr/datasets/liste-des-entreprises-rge-1/
```

Comment:
Recent refacto means you have to use a new format : 
```
datagouv
   - label: Liste des entreprises RGE
      link: https://www.data.gouv.fr/fr/datasets/liste-des-entreprises-rge-1/
```

+ We have to create an uptime robot monitor

Reply:
I keep this one as unresolve till the uptime robot creation id is not done